content/php/concepts/string-functions/terms/html-entity-decode/html-entity-decode.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'PHP html_entity_decode() Function'
+Description: 'Converts HTML entities to characters.'
+Subjects: 
+- 'Web Design'
+- 'Web Development'
+Tags:
+- 'Functions'
+- 'Characters'
+CatalogContent:
+- 'php/concepts/string-functions/terms/html-entity-decode'
+---
+
+
+**'html_entity_decode()'** is the opposite of **'htmlentities()'** in that it converts HTML entities in the string to their corresponding characters.
+
+## Syntax
+```pseudo
+html_entity_decode(str, flags, character-set)
+```
+- 'string - Required. Specifies the string to decode'
+- 'flags - Optional. Specifies how to handle quotes and which document type to use.'
+- c'haracter-set - Optional. A string that specifies which character-set to use.'
+
+## Example
+```php
```

Comment:
Add a small introductory sentence above this code example which can give a rough idea for the below code example.

Reply:
@dakshdeepHERE Resolved.